src/praisonai-agents/praisonaiagents/session.py

```
@@ -580,6 +597,50 @@ def send_message(self, message: str, **kwargs) -> str:
+        if self.is_remote:
+            return  # No cleanup needed for remote sessions
+        
+        # Clear memory
+        if self._memory:
+            try:
+                # Clear short-term and long-term memory for this session
+                # Note: This is a basic implementation - specific memory backends
+                # might need more sophisticated cleanup
+                self._memory = None
+            except Exception:
+                pass  # Ignore cleanup errors
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)